pandas/core/reshape/tile.py

```diff
@@ -26,60 +26,81 @@
 def cut(x, bins, right=True, labels=None, retbins=False, precision=3,
         include_lowest=False):
     """
-    Return indices of half-open bins to which each value of `x` belongs.
+    Bin `x` and return data about the bin to which each `x` value belongs.
+
+    This function splits `x` into the specified number of equal-width half-
```

Comment:
can remove function.

you say 'data' above and use 'information' here. pick one.